storage/s3/s3_test.go

```diff
@@ -3,25 +3,82 @@ package s3_test
 import (
 	"bytes"
 	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"testing"
+	"time"
+
 	"github.com/dreamvo/gilfoyle"
 	"github.com/dreamvo/gilfoyle/config"
 	"github.com/dreamvo/gilfoyle/storage"
-	"github.com/google/uuid"
+	"github.com/ory/dockertest"
+	"github.com/ory/dockertest/docker"
 	assertTest "github.com/stretchr/testify/assert"
-	"io/ioutil"
-	"testing"
-	"time"
 )
 
 func TestS3(t *testing.T) {
 	assert := assertTest.New(t)
```

Comment:
Could you delete this line and use assertions the proper way (like every other tests) please ?

For example, just add the `t` parameter to `assert.NoError(err)` -> `assert.NoError(t, err)`